demo/index.js

```diff
@@ -225,20 +225,26 @@ import { OpenSheetMusicDisplay } from '../src/OpenSheetMusicDisplay/OpenSheetMus
                 return openSheetMusicDisplay.render();
             },
             function(e) {
-                console.warn(e.stack);
-                error("Error reading sheet: " + e);
+                errorLoadingOrRenderingSheet(e, "rendering");
             }
         ).then(
             function() {
                 return onLoadingEnd(isCustom);
             }, function(e) {
-                console.warn(e.stack);
-                error("Error rendering sheet: " + process.env.DEBUG ? e.stack : e);
+                errorLoadingOrRenderingSheet(e, "loading");
                 onLoadingEnd(isCustom);
             }
         );
     }
 
+    function errorLoadingOrRenderingSheet(e, loadingOrRenderingString) {
+        var errorString = "Error " + loadingOrRenderingString + " sheet: " + e;
+        if (process.env.DEBUG) {
```

Comment:
I think nobody will ever do this. But yeah just leave it there for ppl to lazy to pop open the console 👯‍♂️

Reply:
People will never do what? These errors were already printed just like that, only the code was duplicated 4 times. I just made a common error logging method.

Reply:
well, i didn't write that part of the error handling. I'd be fine with removing that part and always printing the stack trace.
Of course then it will also be printed for anyone running the local demo and even our public demo if we don't check the DEBUG env variable, i guess that's what it's for.